.travis.yml

```diff
@@ -26,6 +26,7 @@ jobs:
 
     - name: "npm audit"
       stage: build cache & audit
+      if: tag !~ ^v\d+.* AND commit_message !~ \[skip-audit\]
```

Comment:
The audit step breaks on the release, because it's looking for 1.11.2 when it does the `npm install`. I talked with @mennovandiermen and decided it's best to skip the audit for now when doing the release. 

I've updated https://bugs.earthdata.nasa.gov/browse/CUMULUS-1170 in sprint 19.1.3 to remove this line when we have the lock files in place, which will fix this issue.